src/multimeasurerest.ts

```diff
@@ -227,7 +227,7 @@ export class MultiMeasureRest extends Element {
       x += glyphs[2].width + spacing;
     }
     for (let i = 0; i < n1; ++i) {
-      Glyph.renderGlyph(ctx, x, yTop, rest_scale, rest.glyph_code);
+      if (rest.glyph_code) Glyph.renderGlyph(ctx, x, yTop, rest_scale, rest.glyph_code);
```

Comment:
This looks like a potentially silent bug. Shouldn't there always be a `glyph_code` if we're here?

Reply:
Slash notes do not define a `code_head` this is why `glyph_code` is also optional. I could define `code_head` for slashes as 'N/A' or '' and make the attribute not optional. Would you prefer this approach?

Reply:
code_head no longer optional.